VTEX - Customer Credit API.json

```diff
@@ -7015,8 +7015,8 @@
         "/api/creditcontrol/accounts/{creditAccountId}/transaction": {
             "post": {
                 "tags": ["Account"],
-                "summary": "Create a Pre Authorization",
-                "description": "Authorization hold (also card authorization, pre authorization, or preauth) is the practice that holding this balance as unavailable until either the merchant clears the transaction, also called settlement, or the hold *falls off*.\r\n\r\nThis operation does not appear in statements but the available balance will be updated.\r\n\r\nThe result of this operation is generate a `preauthorizationId` to use in settlement.",
+                "summary": "Create a Pre-Authorization",
+                "description": "Creates a Pre-Authorization. Also known as authorization hold, card authorization or pre-auth, pre-authorization is the practice of holding this balance as unavailable until either the merchant clears the transaction, also called settlement, or the hold *falls off*.\r\n\r\nThis operation does not appear in statements, but the available balance will be updated.\r\n\r\nThe result of this operation is generating a `preauthorizationId` to use in a settlement.",
```

Comment:
O que significa **falls off** nesse contexto?

Reply:
Falls off é um termo comum utilizado quando falamos sobre autorização de captura de valores. Source: https://en.wikipedia.org/wiki/Authorization_hold
"In the case of debit cards, authorization holds can **fall off** the account, thus rendering the balance available again, anywhere from one to eight [business days](https://en.wikipedia.org/wiki/Business_days) after the transaction date, depending on the bank's policy"